doc/content/modules/developer-guide/pages/api-cookbook.adoc

```diff
@@ -0,0 +1,62 @@
+= APIs cookbook
+The APIs cookbook offers a collection of practical examples and step-by-step instructions to help you interact with {product}’s REST APIs.
+It includes ready-to-use code snippets and explanations, allowing you to integrate with other tools, automate tasks, and explore advanced features efficiently.
+Each recipe aims for clarity, ease of use, and customizability for different use cases.
+
+== Setup instructions
+To get started with the examples, Python is used to interact with {product}’s features and data.
+Python offers a powerful way to automate tasks, handle API calls, and process data, making it an ideal choice for this purpose.
+Follow the steps after to set up your environment and get everything ready to run the provided code snippets.
+
+=== Step 1: Install required Python libraries
+Confirm you have the following Python libraries installed before running the code snippets:
+
+* `requests`: For making HTTP requests to the {sysmlv2} API.
+* `pandas`: For organizing and manipulating data.
+* `json`: For handling JSON responses.
```

Comment:
`pip install json` do not work, `json` is not recognized as a module.
So is it relevant to put it in this list?

Reply:
Fixed